lib/helper/WebDriverIO.js

```diff
@@ -992,18 +992,23 @@ class WebDriverIO extends Helper {
   /**
    * {{> ../webapi/waitForVisible }}
    */
-  waitForVisible(locator, sec) {
+  waitForVisible(locator, sec, reverse) {
     sec = sec || this.options.waitForTimeout;
-    return this.browser.waitForVisible(withStrictLocator(locator), sec * 1000);
+    return this.browser.waitForVisible(withStrictLocator(locator), sec * 1000, reverse);
   }
 
   /**
-   * Waits for an element to become invisible on a page (by default waits for 1sec).
-   * Element can be located by CSS or XPath.
+   * {{> ../webapi/waitForInvisible }}
    */
-  waitToHide(locator, sec) {
```

Comment:
`waitToHide` was nice. Maybe we are ok to name method like this? Also some people may use it so it's not a good idea to remove it completely